### PR TITLE
Add travis.yml

### PR DESCRIPTION
**Pull request by uttu357**
- Run tests for both Django and Node.js applications

The build is failing because of errors in frontend build script which are resolved by #4

### CODE REVIEW

**Comment by tocttou** [comment]
Are you sure you want to use python2 here?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - nvm use 6.0.0
+  - npm install -g yarn@0.23.4
+
+install:
+  - pip install -r requirements/dev.txt
+  - cd frontend && yarn install
+  - cd ../
+
+script:
+  - flake8 cloudcv/
```
Comment (deshraj) [inline .travis.yml]: flake8 should also be run on the `apps/` directory, right? Can we make it `flake8 ./`?

Comment (uttu357) [inline .travis.yml]: It then starts to also run on frontend/ directory. We can add apps/ directory to travis.yml once we create it..

**Comment by deshraj** [comment]
No, I don't think that should be a problem since it only works on python files and won't affect the js files. Anyways, you can ignore the `frontend/` directory by setting the flake8 configuration in setup.cfg file. For reference, see this https://github.com/Cloud-CV/EvalAI/blob/master/setup.cfg

**Comment by uttu357** [comment]
Sure will do the required! :+1:

**Comment by deshraj** [comment]
Looks good. Merging this. :)

**Pull request merged**